tests/test_models.py

```diff
@@ -85,6 +88,12 @@ def test_last_theme_activated_on_multiple_themes_activated(self):
                 self.assertEqual(Theme.get_active_theme().pk, theme.pk)
         self.__test_active_theme()
 
+    def test_use_db_defined_in_kwargs(self, get_active_theme):
+        Theme.objects.all().delete()
+        theme_1 = Theme.objects.create(name="Custom 1", active=True)
+        kwargs = {"db" : "default"}
+        Theme.get_active_theme(**kwargs)
+
```

Comment:
This test is ok, but I think it would have passed the tests even before the other changes you made because it uses the default database. Could you add also another that uses another database please?

Reply:
Just added another  test for failure

Reply:
If you could add a third test for multi-db it would be great!